README.md

```diff
@@ -197,7 +197,11 @@ Run `yarn license-report` to generate a license report for all npm packages. See
 
 ### Classic Mode
 
-This is first release of the DUS solution. The major services included in this mode include Amazon Elasticsearch, Amazon Textract, Amazon Comprehend and Amazon Comprehend Medical that allow digitization, information extraction and indexing in DUS.
+This is first release of the DUS solution. The major services included in this mode include Amazon Elasticsearch, Amazon Textract and Amazon Comprehend that allow digitization, information extraction and indexing in DUS.
+
+### Comprehend Medical Enabled Mode
+In comprehend medical mode, DUS provides the capability to extract medical information in the documents in addition to the capabilities of the classic mode.
```

Comment:
should we make this more descriptive?

In the Comprehend Medical Enabled mode, DUS provides the capability to extract medical information such as Medical Entities, ICD-10 and RX Norm onotlogies from uploaded documents

Reply:
Makes sense. Will update,